assets/js/app/Instance.js

```diff
@@ -8,8 +8,8 @@
 
   LocusZoom.Instance Class
 
-  An instance is an independent LocusZoom object. Many instances can exist simultaneously
-  on a single page, each having its own data caching, configuration, and state.
+  An Instance is an independent LocusZoom object. Many such LocusZoom objects can exist simultaneously
```

Comment:
Maybe we can change it to "Plot" in another pull request and see how we like it.